packages/composite-checkout/src/components/checkout-steps.tsx

```diff
@@ -228,6 +228,7 @@ export function Checkout( {
 
 	const classNames = joinClasses( [
 		'composite-checkout',
+		'fs-exclude',
```

Comment:
Let's add a comment here to explain that this is needed for the privacy settings in our FullStory integration.